cwms-data-api/src/test/java/cwms/cda/api/TimeSeriesGroupControllerTestIT.java

```diff
@@ -583,4 +651,313 @@ void test_add_assigned_locs() throws Exception {
             .log().ifValidationFails(LogDetail.ALL,true)
             .statusCode(is(HttpServletResponse.SC_NO_CONTENT));
     }
+
+    @Test
+    void test_patch_permissions_CWMS() throws Exception {
+        ObjectMapper mapper = new ObjectMapper();
+
+        InputStream resource = this.getClass().getResourceAsStream(
+                "/cwms/cda/api/timeseries_permission_create4.json");
```

Comment:
Please provide a more meaningful name for these. I tried to skip directly to them to get a sense of data and got confused.

Reply:
same

Reply:
Renamed to timeseries_create_SPK, hopefully this is a bit clearer

Reply:
The confusion is the numbering. The time series all look similar, you could just edit the id at runtime to get your multiple ids in the group.

Reply:
I reduced the number of test JSON files from 4 to 1 with TSID replacement to differentiate the Timeseries names